email.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+
+# If running locally (i.e. not on CI), fetch the email from the users git config
+if [ $CI != 'true' ]
+then
+    echo "CI deploy not detected. Using local variables instead"
+    # Fetch the variables from the git config.
+    AUTHOR_EMAIL=$(git config user.email)
+    REACT_APP_DEPLOYS_EMAIL=$REACT_APP_DEPLOYS_EMAIL
+    TASK_NAME=$(git rev-parse HEAD)
+fi
+
+# Validate necessary variables are set
+if [ '$REACT_APP_DEPLOYS_EMAIL' == '' ]
+then
+    echo "REACT_APP_DEPLOYS_EMAIL is not set"
+    exit 1
+fi
+
+if [ '$AUTHOR_EMAIL' == '' ]
+then
+    echo "AUTHOR_EMAIL is not set"
+    exit 1
+fi
+
+if [ '$TASK_NAME' == '' ]
+then
+    echo "Was unable to fetch the task_name or git commit associated with this build"
+    exit 1
+fi
+
+# Fetch previous release tag and get the commits since that tag
+PREVIOUS_VERSION=$(git describe --tags  --abbrev=0  `git rev-list --tags --max-count=1 --skip=1`)
+git log --no-merges $PREVIOUS_VERSION..HEAD --pretty="format:%s (%h)" > body.txt
+
+# Determine which verson of evergreen is available and use that
+if ! [ -x "$(command -v evergreen)" ]
+then
+  echo 'Error: evergreen is not on $PATH.'
```

Comment:
Nit: I think this means there will be a line that says "Error" in every CI deploy, which is maybe misleading.